src/Symfony/Component/Notifier/Bridge/Bluesky/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ---
 
  * Add option to attach a media
+ * [BC Break] Change the returned message ID from record's 'cid' to 'uri'
```

Comment:
Should be reverted as changelogs are auto-updated for bugfixes

Reply:
Only the main CHANGELOG. Here, we want to signify something unusual, I think it's fine.

Reply:
Fine for me as well, but I think it's worth noticing that we had similar discussions in the past and the conclusion was it's either a bugfix or a BC break, not both.